src/components/Sections/VideoSection/index.tsx

```diff
@@ -0,0 +1,54 @@
+import BaseComponent from "@/components/BaseComponent";
+import LineSeparator from "@/components/LineSeparator";
+import Headline from "@/components/Headline";
+import { VideoSectionProps } from "@/types/sections";
+import { Component, Prop } from "vue-property-decorator";
+
+@Component({
+  name: "VideoSection",
+})
+class VideoSection extends BaseComponent<VideoSectionProps> {
+  @Prop() youtubeId!: VideoSectionProps["youtubeId"];
```

Comment:
This property should be marked as required

Reply:
done